backend/handlers/markets/resolvemarket.go

```diff
@@ -90,6 +90,7 @@ func ResolveMarketHandler(w http.ResponseWriter, r *http.Request) {
 	market.IsResolved = true
 	market.ResolutionResult = resolutionData.Outcome
 	market.FinalResolutionDateTime = time.Now()
+	// TODO: Do we also need to set market.ResolutionDateTime here?
```

Comment:
Please see my comments describing the fields in more detail on this MR within the comments section, not sure if that helps?

In general, rather than creating a TODO comment I think a better practice would be to create a ticket, correct? Was this comment intended to be temporary or were you thinking leave a to-do here in the code?

Reply:
Agreed, my intent was just to draw attention to a place I was considering a change